test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -358,6 +358,21 @@ function (Q, OktaAuth, WidgetUtil, Okta, Util, AuthContainer, IDPDiscoveryForm,
           expect(test.form.helpLinkHref()).toBe('https://bar.com');
         });
       });
+      itp('has helpFooter with right aria-attributes and default values', function () {
+        return setup().then(function (test) {
+          expect(test.form.helpFooter().attr('aria-expanded')).toBe('false');
+          expect(test.form.helpFooter().attr('aria-controls')).toBe('help-links-container');
+        });
+      });
+      itp('helpFooter toggles aria-expanded correct', function () {
+        return setup().then(function (test) {
+          expect(test.form.helpFooter().attr('aria-expanded')).toBe('false');
+          test.form.helpFooter().click();
+          return tick(test);
```

Comment:
is this to wait for the animation to complete?

Reply:
Yes.

Reply:
See comment below. Basically, trying to avoid using tick().